File: app/src/main/java/dev/ragnarok/fenrir/api/model/response/MessageHistoryResponse.java
package dev.ragnarok.fenrir.api.model.response;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import dev.ragnarok.fenrir.api.model.VKApiCommunity;
import dev.ragnarok.fenrir.api.model.VKApiConversation;
import dev.ragnarok.fenrir.api.model.VKApiMessage;
import dev.ragnarok.fenrir.api.model.VKApiUser;

public class MessageHistoryResponse {

    @Nullable
    @SerializedName("items")
    public List<VKApiMessage> messages;

    @SerializedName("count")
    public int count;

    @SerializedName("unread")
    public int unread;

    @Nullable
    @SerializedName("conversations")
    public List<VKApiConversation> conversations;

    @Nullable
    @SerializedName("profiles")
    public List<VKApiUser> profiles;

    @Nullable
    @SerializedName("groups")
    public List<VKApiCommunity> groups;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/response/MessageHistoryResponse.kt
package dev.ragnarok.fenrir.api.model.response

import com.google.gson.annotations.SerializedName
import dev.ragnarok.fenrir.api.model.VKApiCommunity
import dev.ragnarok.fenrir.api.model.VKApiConversation
import dev.ragnarok.fenrir.api.model.VKApiMessage
import dev.ragnarok.fenrir.api.model.VKApiUser

class MessageHistoryResponse {
    @SerializedName("items")
    var messages: ArrayList<VKApiMessage>? = null

    @SerializedName("count")
    var count = 0

    @SerializedName("unread")
    var unread = 0

    @SerializedName("conversations")
    var conversations: List<VKApiConversation>? = null

    @SerializedName("profiles")
    var profiles: List<VKApiUser>? = null

    @SerializedName("groups")
    var groups: List<VKApiCommunity>? = null
}